lua/core/nvimtree.lua

```diff
@@ -88,15 +89,19 @@ M.setup = function()
   end
 
   vim.cmd "au WinClosed * lua require('core.nvimtree').on_close()"
+
+  if lvim.builtin.nvimtree.on_config_done then
+    lvim.builtin.nvimtree.on_config_done(nvim_tree_config)
+  end
 end
 
-M.on_open = function()
+function M.on_open()
```

Comment:
why did we make these functions different from the rest of the code
everywhere else we have `M.sth = function()` but in this file we have `function M.sth()`

Reply:
Arf, there is a big style mix in our code base, I think we should go for this syntax as the other one would be more appropriate for defining lambdas. There was a quick discussion and that's what some of us agreed for (maybe only kylo and me actually :smile: ), but since it's not been smth official, you probably never heard of it.

Reply:
sure we can take of this in another PR then